presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestTypedSet.java

```diff
@@ -130,9 +134,29 @@ public void testBigintSimpleTypedSet()
         }
     }
 
+    @Test
+    public void testMemoryExceeded()
```

Comment:
so you're suggesting one very large element vs many elements. 

both should work, but somehow when I tried a single large element in the more targeted unit test, it did not cause failure. 

I'll revisit